src/vs/server/node/remoteAgentEnvironmentImpl.ts

```diff
@@ -356,7 +356,6 @@ export class RemoteAgentEnvironmentChannel implements IServerChannel {
 		return scannedExtensions.map(e => toExtensionDescription(e, false)).filter(ext => {
 			// TODO: remove this once whe decoupled gitpod extensions from gp-code
 			const ignoreExtensions = [
-				'vscode.github-authentication',
 				'gitpod.gitpod-shared',
 				'gitpod.gitpod-remote-ssh',
 				'gitpod.gitpod-desktop'
```

Comment:
```suggestion
				'gitpod.gitpod-remote-ssh'
```

Reply:
it's back again

Reply:
ah, I misunderstood! I thought we had to remove only the first one 😂

Reply:
let me squash